tests/test_coreg.py

```diff
@@ -822,7 +821,7 @@ def test_warp_dem():
     # The warped DEM should have the value 'elev_shift' in the upper left corner.
     assert warped_dem[0, 0] == elev_shift
     # The corner should be zero, so the corner pixel (represents the corner minus resolution / 2) should be close.
-    assert warped_dem[-1, -1] < 1.0
+    assert warped_dem[-2, -1] < 1
```

Comment:
Why -2?

Reply:
Because the last row is NaNs propagated by the warp function. I added a comment to specify it in the test.